tests/test_material.py

```diff
@@ -222,13 +222,22 @@ def test_activity(self):
         assert_equal(set(obs.values()), set(exp.values()))
 
 
-    def test_decay_heat(self):
+    def test_decay_heat_stable(self):
         mat = Material({922350000: 0.05, 922380000: 0.95}, 15)
         obs = mat.decay_heat()
         exp = {922350000: 4.48963565256e-14, 922380000: 1.2123912039e-13}
         assert_equal(set(obs), set(exp))
         for key in exp:
             assert_almost_equal(obs[key], exp[key])
+    
+
+    def test_decay_heat_metastable(self):
+        mat = Material({471080001: 0.5, 551340001: 0.5}, 2)
+        obs = mat.decay_heat()
+        exp = {471080001: 7.33578506845e-8, 551340001: 6.241109861949e-3}
```

Comment:
Just one quick though, how did you get those number ?
did you run a calculation using the fixed Pyne code ?
If so it might be good to have an other way to check it ? (not sure it is easy to find..)
in any case, I think you should put a comment to explain how those number where determined.

Reply:
I copied the decay heat values from the jupyter notebook that I created to test some isotopes: [test_decay_heat_fix](https://gist.github.com/aruzzo/3e87e6381702b6572b13d31e43429c26). In short, I copied Avogadro's number and the value of MeV per MJ from pyne's data.cpp file and I rewrote the equation for calculating decay heat in python following the same steps in pyne's material.cpp file. For the atomic data, I got the info from outside sources (which should also be the same sources that pyne uses) in order to separate the calculation from pyne code. The half-lives were from [BNL](https://www.nndc.bnl.gov/nudat2/chartNuc.jsp), the q-values were from [ORNL](https://technicalreports.ornl.gov/cppr/y2001/rpt/97914.pdf), and the atomic masses were from [AMDC](https://www-nds.iaea.org/amdc/ame2016/mass16.txt).

Reply:
great ! could you add a small comment to explain that ?
something like:
`// external calculation using Half-live: BNL, q-values: ORNL, Atomic mass: AMDC`